src/journal/journalctl.c

```diff
@@ -1452,23 +1458,36 @@ static int list_boots(sd_journal *j) {
         if (count == 0)
                 return count;
 
-        pager_open(arg_pager_flags);
+        table = table_new("#", "boot id", "first entry", "last entry");
```

Comment:
`#` is not a good label especially for json. How about to use `"index"` or `"boot index"` at least when printed in json format?

Reply:
Would "idx" work when used in both cases?

```
$ build-san/journalctl --list-boots --file boot-test.journal 
IDX BOOT ID                          FIRST ENTRY                 LAST ENTRY                 
-3  39d66eb1925f4d01b8464d502650a714 Sat 2022-03-05 15:20:33 CET Sat 2022-03-05 16:19:21 CET
-2  5dffeb08a27344d5ae9e2fc244bbcbc5 Fri 2022-03-11 17:23:57 CET Fri 2022-03-11 17:38:31 CET
-1  c8ebd52915b642c39eda4bf00f864f79 Fri 2022-03-11 17:38:41 CET Fri 2022-03-11 20:03:46 CET
 0  00bcba97c7094fa88cc5d1cf2a389057 Sat 2022-03-12 20:39:08 CET Sat 2022-03-12 20:40:18 CET

$ build-san/journalctl --list-boots --file boot-test.journal -o json-pretty
[
        {
                "idx" : "-3",
                "boot_id" : "39d66eb1925f4d01b8464d502650a714",
                "first_entry" : 1646490033438495,
                "last_entry" : 1646493561047353
        },
        {
                "idx" : "-2",
                "boot_id" : "5dffeb08a27344d5ae9e2fc244bbcbc5",
                "first_entry" : 1647015837289036,
                "last_entry" : 1647016711595489
        },
        {
                "idx" : "-1",
                "boot_id" : "c8ebd52915b642c39eda4bf00f864f79",
                "first_entry" : 1647016721056382,
                "last_entry" : 1647025426397414
        },
        {
                "idx" : "0",
                "boot_id" : "00bcba97c7094fa88cc5d1cf2a389057",
                "first_entry" : 1647113948506002,
                "last_entry" : 1647114018943637
        }
]
```

I mean, there's an "extra" space in the "IDX" column (since all indexes in my test journal are two-chars wide at most), but it uses the same _label_ in both cases.

Reply:
Still I am not a fan of abbreviated name in json format. `idx` or `#` is OK for table format. How about the following?
```
table = table_new(OUTPUT_MODE_IS_JSON(arg_output) ? "index" : "idx", "boot id", "first entry", "last entry");
```